src/tasks/processIssueComment/probot-processIssueComment.js

```diff
@@ -24,6 +24,13 @@ async function processAddContributor({
     contributions,
     branchName,
 }) {
+    if (!contributions.length) {
```

Comment:
`if (!contributions.length > 0) {`

Reply:
If it needs to be more explicit, than `!contributions.length`, wouldn't `contributions.length === 0` be crystal clear while having the same effect as `!contributions.length`?

Reply:
Yep that would be clear too!

This is a habit from my C days where i'd never use == 0. Totally acceptable here!

Reply:
Ah, I see!